guides/plugins/plugins/framework/flow/add-flow-builder-action.md

```diff
@@ -103,7 +103,7 @@ class CreateTagAction extends FlowAction
 
         $tags = $config['tags'];
 
-        // just a step to make sure you're dispatching correct action
+        // just a step to make sure you're dispatching the correct action
```

Comment:
Please be aware to not use contractions such as you're, we're, let's, it's. But you are free to use negative contractions such as don't, can't, etc... 
We are in the process to publish documentation guidelines to keep all the contributors aware of certain writing styles.

```suggestion
        // just a step to make sure you are dispatching the correct action
```

Reply:
Duly noted, i will make corrections.